### ISSUE TITLE
gulp task scripts unable to uglify in production build

### ISSUE THREAD

**Issue by jobayerarman** [jobayerarman/jobayerarman.github.io#21]
```bash
TASK: [gulp-uglify]
PROB: unable to minify JavaScript
FILE: H:\Temp\MyWorkshop\Development\Web\jobayerarman.github.io\src\js\user\user
.min.js
```

**Comment by jobayerarman** [comment]
#### error file
```
FILE: H:\Temp\MyWorkshop\Development\Web\jobayerarman.github.io\src\js\user\user
.min.js
```
#### error line
`Line: 194`
```javascript
init: () => {
```

make uglify ES6 compatible

**Comment by jobayerarman** [comment]
There are no supporting uglify(minify) tools for ES6 Syntax. 
gulp task after [gulp-babel](https://www.npmjs.com/package/gulp-babel) compile (es6 -> es5)
#### install package dev
```
yarn add -D gulp-babel
```
#### update gulp task
```javascript
var babel  = require('gulp-babel');

gulp.src( scripts.user.src.files )
    .pipe( plumber({errorHandler: errorLog}) )
    .pipe( babel({ presets: ['es2015'] }) )
    .pipe( concat( scripts.user.dest.filename ) )
    .pipe( gulpif( config.production, uglifyScripts() ) )

    .pipe( gulp.dest( scripts.user.dest.path ) )

    .pipe( size({
      showFiles: true
    }) );
```

**Comment by jobayerarman** [comment]
fixed in commit fd922219d6b2241a7dd1140a22bdcbd3b2f0697c
